### PR TITLE
S5 product cards

### PR DESCRIPTION
**Pull request by tarasivanchyshyn**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ import { dropdownsHeaders } from 'mockedData';
 import classes from './Header.module.scss';
 
 function Header() {
-  const { active } = classes;
+  // const { active } = classes;
 
-  const setActive = ({ isActive }: { isActive: boolean }) =>
-    isActive ? active : '';
+  // const setActive = ({ isActive }: { isActive: boolean }) =>
+  //   isActive ? active : '';
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: What's this ?

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ function Header() {
         </div>
       </div>
 
-      <nav>
+      {/* <nav>
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: The same

File: src/components/Products/Main/ProductCards/Card/Card.module.scss [hunk 3]
```diff
@@ -0,0 +1,181 @@
+@import 'variables';
+
+.card {
+  border: 1px solid #d1d1d1;
```
Comment (Yuliia143) [inline src/components/Products/Main/ProductCards/Card/Card.module.scss]: Where should the color be ?

File: src/components/Products/Main/ProductCards/Card/Card.module.scss [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    height: 100%;
+  }
+
+  &__image {
+    display: block;
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+    border-radius: 12px;
+    background: #a9a9a9;
```
Comment (Yuliia143) [inline src/components/Products/Main/ProductCards/Card/Card.module.scss]: color x2

File: src/components/Products/Main/ProductCards/Card/Card.module.scss [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    border: 2px solid $card-btnWish-bg;
+    border-radius: 12px;
+
+    span {
+      color: $black;
+    }
+
+    &:hover {
+      background: #e9e9e9;
+      border-color: #e9e9e9;
```
Comment (Yuliia143) [inline src/components/Products/Main/ProductCards/Card/Card.module.scss]: color x3

File: src/components/Products/Main/ProductCards/Card/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+import classes from './Card.module.scss';
+
+interface CardProps {
+  product: IProduct;
+}
+
+const Card: FC<CardProps> = ({ product }) => {
+  const { title, description, image, rating, freshness, farm } = product;
+  const { delivery, units, stock, price, priceBefore } = product;
+  const { freeShipping, deliveryTime } = product;
```
Comment (Yuliia143) [inline src/components/Products/Main/ProductCards/Card/Card.tsx]: Don't like, I think nobody does this in practice

File: src/components/Products/Main/ProductCards/ProductCards.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    data: products,
+    isLoading,
+    error
+  } = productsAPI.useFetchAllProductsQuery(2);
+
+  return (
+    <div className={classes.cards}>
+      <ul>
+        {isLoading && <h1>Downloading...</h1>}
+        {error && <h1>Error occured</h1>}
```
Comment (Yuliia143) [inline src/components/Products/Main/ProductCards/ProductCards.tsx]: Could be before ul

File: src/components/Footer/ContactsColumn/ContactsColumn.module.scss [hunk 8]
```diff
@@ -6,20 +6,13 @@
...
-    @include poppins;
     margin: 0 0 1rem;
-    font-weight: 600;
-    font-size: 1.125rem;
-    line-height: 1.687rem;
-    cursor: default;
+    font: 600 1.125rem/1.687rem $poppins;
   }
 
   .contacts__contact {
```
Comment (Kost927) [inline src/components/Footer/ContactsColumn/ContactsColumn.module.scss:9]: .contacts {
&__header {};
&__contact {};
}

Comment (tarasivanchyshyn) [reply src/components/Footer/ContactsColumn/ContactsColumn.module.scss:9]: but .contacts is in different component and i cant use & at top

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**